packages/@stylexjs/postcss-plugin/__tests__/index-test.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const path = require('path');
+const postcss = require('postcss');
+const createPlugin = require('../src/plugin');
+
+describe('@stylexjs/postcss-plugin', () => {
+  const fixturesDir = path.resolve(__dirname, '__fixtures__');
+
+  async function runStylexPostcss(options = {}, inputCSS = '@stylex;') {
+    // Create a new instance for each test as the plugin is stateful
+    const stylexPostcssPlugin = createPlugin();
+
+    const plugin = stylexPostcssPlugin({
+      cwd: fixturesDir,
+      include: ['**/*.js'],
+      babelConfig: {
+        configFile: path.join(fixturesDir, '.babelrc.json'),
+      },
+      ...options,
+    });
+
+    const processor = postcss([plugin]);
+    const result = await processor.process(inputCSS, {
+      from: path.join(fixturesDir, 'input.css'),
+    });
+
+    return result;
+  }
+
+  it('extracts CSS from StyleX files', async () => {
```

Comment:
```suggestion
  test('extracts CSS from StyleX files', async () => {
```

Use `test` for all these tests please

Reply:
Fixed in https://github.com/facebook/stylex/pull/1067/commits/a73a2c9dd0d87b44a7a8a38f75d2cef2d237063b